README.md

```diff
@@ -44,6 +44,7 @@ Follow the setup recommendations of your cloud provider.
 - [Contributing](CONTRIBUTING.md)
 
 ## Talks
+- [How To Auto-Scale Kubernetes Clusters With Karpenter](https://youtu.be/C-2v7HT-uSA)
```

Comment:
This is getting a little bit hard to parse without some organization/categorization. 

Can you put a date on this and the other ones, e.g. `[12/20/21] How To Auto-Scale Kubernetes Clusters With Karpenter`

Do you mind back